index.js

```diff
@@ -59,7 +59,7 @@ module.exports = function(fn) {
 
     res.end = function(chunk, encoding, cb) {
       debug('end called');
-      var args = Array.prototype.slice.call(arguments, 1);
+      var args = Array.prototype.slice.call(arguments);
```

Comment:
strange we didn't have a case to catch see this error when no intercept method was provided?